app/app/constants.ts

```diff
@@ -3,7 +3,7 @@ export const WYOMING_PORT = 10700;
 export const STORAGE_KEY_UUID = 'zeroconfuuid';
 export const STORAGE_KEY_RUN_BACKGROUND_TASK = 'bgtaskshouldrun';
 export const STORAGE_KEY_SAVED_SETTINGS_PROTO = 'savedsettingsproto';
-export const APP_VERSION = '0.9.4';
+export const APP_VERSION = '0.9.5';
```

Comment:
Can you change the `VERSION` file and then run `scripts/update_vresion.sh` ? That makes sure that all of the various places where the version number appears stay in sync.

I'd highly recommend installing the [pre-commit](https://pre-commit.com/) hooks in your dev environment, which will do this for you :)

Reply:
Done